app/views/text_components/_text_component_row.html.haml

```diff
@@ -3,6 +3,9 @@
   %td= text_component.from_day
   %td= text_component.to_day
   %td
+    - if status_class_mapping = {'draft' => 'default', 'fact_checked' => 'warning', 'published' => 'success'} and status_class_mapping.key? text_component.publication_status
```

Comment:
Okay, I can understand that this wasn’t the most elegant way to solve this (and I have moved this to a model method instead to avoid cluttered views). But let me ask one more question just to make sure I understand this correctly… ;)

Is there anything wrong about using `and` in this specific example (other than the reason stated above)? I am actually using it to chain those two expressions, right?